assets/static/css/style.css

```diff
@@ -0,0 +1,75 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  h1 {
+   text-align: center;
+  }
+  
+  input[type=text], select, textarea {
+    width: 100%;
+    padding: 12px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    resize: vertical;
+  }
+  
+  label {
+    padding: 12px 12px 12px 0;
+    display: inline-block;
+  }
+  
+  .primary-button {
+    background-color: #115debc8;
+    color: white;
+    padding: 12px 20px;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+    float: right;
+  }
+  
+  .primary-button:hover {
+    background-color: #115deb;
+  }
+
+  .secondary-button {
+    background-color: #d1d3d87b;
+    color: black;
+    padding: 12px 20px;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+    float: right;
+  }
+  
+  .secondary-button:hover {
+    background-color: #d1d3d8c8;
+  }
+
+  .container {
+    border-radius: 5px;
+    background-color: #f2f2f2;
+    padding: 20px;
+    margin: auto;
+    width: 50%;
+  }
+  
+  .col-25 {
```

Comment:
I would rather see us use flexbox (https://css-tricks.com/snippets/css/a-guide-to-flexbox/)